apps/portal/README.md

```
@@ -46,33 +46,33 @@ Refer the [docs](https://ghost.org/help/setup-members/#customize-portal-settings
 ```
 
 This starts all frontend apps (including Portal.)
 ---
 
-To run Portal in a standalone fashion, use `yarn start` and open [http://localhost:3000](http://localhost:3000).
+To run Portal in a standalone fashion, use `pnpm start` and open [http://localhost:3000](http://localhost:3000).
 
 ## Build
 
 To create a production minified bundle in `umd/portal.min.js`:
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)